### PR TITLE
added documentation for converting a non-container app into mystikos

### PR DESCRIPTION
**Pull request by salsal97**

### CODE REVIEW

**Comment by asvrada** [comment]
Should we mention this doc in place like readme or user-getting-started.md

File: doc/user-converting-existing-binary-into-mystikos-app.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+The final step in this process would be to create the container needed for the appbuilder. Coherently arrange the statements above so that: 
+1. You have your base OS included
+2. Include any dependencies required for your binary
+3. Include any ENV VARs needed for invokation
+4. Copy the binary you want to execute in mystikos from your host system into the container
+
+You can build and run the container app with the following command to make sure it's correct: \
+`docker run $(docker build -q .) /cmd/to/run/binary <args-if-any>`
+
+The process to create the container is simple as defined by docker. You can simply let us handle it from there by invoking -
```
Comment (asvrada) [inline doc/user-converting-existing-binary-into-mystikos-app.md:68]: `The process to create the container is simple as defined by docker`
I am not sure what this line is trying to tell, could you rephrase it a bit?

**Review by asvrada — APPROVED** [review]
Minor comments but LGTM

**Review by mikbras — APPROVED** [review]

File: doc/user-converting-existing-binary-into-mystikos-app.md (hunk 1 above)
Comment (salsal97) [reply doc/user-converting-existing-binary-into-mystikos-app.md]: Hey, I added a comma here that I think I needed. Thanks for pointing that out!

Also - mentioned the doc in the getting started readme
